### PR TITLE
Add MetadataText from check-ui to check-web

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

- Adding a version of `MetadataText` to check-web from check-ui to uncouple that connection and allow MetadataText to more easily use CDS styles.
- Remove unnecessary expand/collapse functionality on annotations settings page
- Update error flash message with some CDS buttons instead of material

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

updates to integration tests

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by amoedoamorim** [comment]
Tried this locally and :LGTM: so far

**Comment by brianfleming** [comment]
> Tried this locally and :LGTM: so far

Thanks for taking a look @amoedoamorim when I get the test to behave I will put it up for review, then try to work my way through the other components from check-ui

**Comment by brianfleming** [comment]
ok @amoedoamorim I cleaned up the tests and made some other random CDS style related changes while waiting, please to re-review

**Comment by brianfleming** [comment]
Reviewers FYI, I am not going to merge this until after the release tonight, just want to get it all buttoned up before that

File: src/app/components/task/Task.js [hunk 1]
```diff
@@ -458,21 +476,25 @@ class Task extends Component {
...
-            />
-          </Button>
+            label={
+              <FormattedMessage
+                id="metadata.delete"
+                defaultMessage="Delete"
+                description="This is a label that appears on a button next to an item that the user can delete. The label indicates that if the user presses this button, the item will be deleted."
+              />
+            }
+          />
```
Comment (amoedoamorim) [inline src/app/components/task/Task.js:497]: Let's replace these metadata messages with global edit, cancel, delete & save?

**Review by amoedoamorim — APPROVED** [review]

File: src/app/components/task/Task.js (hunk 1 above)
Comment (brianfleming) [reply src/app/components/task/Task.js:497]: ooh, good catch. One day I will see this before you do and do it automatically...but will update!

**Pull request merged**